meetings/2024-11-12-taskforce-meeting.md

```diff
@@ -46,33 +46,37 @@ Original agenda: https://github.com/swicg/activitypub-trust-and-safety/issues/26
 
 ## Minutes
 
-Reminder of process, W3C account requirements and contributor agreements (links in agenda)
+**Please:** Make sure you have a W3C Account, and have joined the [Social Web CG](https://www.w3.org/community/socialcg/join)
 
 Code of conduct: this is not a group to work directly on moderation issues, nor is it about moderation policies of admins or platforms, but to work on the systems & protocols that can support moderation.
 
+**Please:** make sure you read the Taskforce's [Code of Conduct](https://github.com/swicg/activitypub-trust-and-safety/blob/main/CODE_OF_CONDUCT.md) and the linked W3C documents.
+
 Introductions from the above people.
 
 ### Meeting times and cadence
 
-RESOLVED: 5pm CET Tuesdays (the time of this meeting) is OK for recurring meetings.  
+**RESOLVED:** 5pm CET Tuesdays (the time of this meeting) is OK for recurring meetings.
 
 Evan reminded participants to use GitHub to help folks who have to participate asynchronously may do so. 
 Staging process will be to be followed: https://github.com/swicg/potential-charters/blob/main/stage-process.md
 
-Proposal: Meeting cadence every two weeks (same time)
+**Proposal:** Meeting cadence every two weeks (same time)
+
+**RESOLVED:** Meeting cadence every two weeks (same time)
 
-RESOLVED: Meeting cadence every two weeks (same time)
+@dmitrizagidulin will setup the recurring meeting on the Social Web CG calendar.
 
 ### Discussion of Scope
 
 Evan asked about bundling things in the scope together?  Emelia explained that Trust and Safety can have requirements on a number of documents, many of which might not be "owned" by the Trust and Safety TF.  E.g. Reply-control is being defined elsewhere but we can make recommendations to implementors about it. Similarly we aren't defining the S2S HTTP message signatures, but we can make a recommendation to use that.  
 
 The scope proposed (in the agenda and at the link in the agenda) is roughly in priority order or order of work. The Flag mechanisms are somewhat under-specified.  We can improve those mechanisms.  Some examples:
- * Mastodon only accepts 500 characters in a report text, but misskey can send ~4000.  
- * Mastodon expects Actor in Flag reports although the spec isn't clear about requiring that, so flags without actor get dropped.  
+ * Mastodon only accepted 500 characters in a report text, but misskey can send ~4000, this lead to interoperability issues (and has since been fixed).  
+ * Mastodon expects Actor in the list of `objects` in a Flag Activity, although the spec doesn't requiring that, so Flags without the actor in the `objects` list get dropped.
 
 Dan: something that might be useful, in the TAG we've been developing the [privacy principles documentd](
```

Comment:
nitpick -> documented

Reply:
whoops, missed that. I think it's clear enough though, so no need to do another PR now that dmitri has merged this one.